lab-bundle-spec.md

```diff
@@ -335,17 +341,18 @@ Note that a lab will only launch on a given deployment if the deployment has an
 ###### Valid custom property references
 
 The valid `reference`s for an `aws_account` resource are:
-
-- [AWS_ACCOUNT].account_number
-- [AWS_ACCOUNT].username
-- [AWS_ACCOUNT].password
-<!-- Legacy display option replacements -->
-- [AWS_ACCOUNT].sts_link
-- [AWS_ACCOUNT].access_key_id
-- [AWS_ACCOUNT].console_url
-- [AWS_ACCOUNT].rdp_credentials
-- [AWS_ACCOUNT].ssh_credentials
-- [AWS_ACCOUNT].vnc_link
+<!-- Legacy display option replacements after password resource reference-->
```

Comment:
This HTML comment doesn't make sense up here. I don't think it's super important to preserve so I'd recommend deleting it.

Reply:
removed